### DISCUSSION TITLE
Sorting the insights dependency graph lists

### DISCUSSION THREAD

**Discussion by shapkarin** [community/community#5575] [Repositories]
## Can be useful for dependents repositories and packages lists
That really interesting to look at dependents repositories and find some great project. Some popular repos has a lot of dependents so I wonder be able to sort them by stars or forks.

## Current roadmap
There is no any projects tasks at [github/roadmap](https://github.com/github/roadmap/projects/1?card_filter_query=insights) but I found some [issues](https://github.com/github/roadmap/issues?q=is%3Aissue+Insights+)

**Comment by andriyor** [comment]
You can use this tool [github-tooling/ghtopdep: CLI tool for sorting dependents repo by stars](https://github.com/github-tooling/ghtopdep)

Issue in community repository: [Allow dependents to be sorted by stars · Issue #1537 · isaacs/github](https://github.com/isaacs/github/issues/1537)

**Reply by shapkarin** [reply]
nice. hope to see the same at GitHub API and UI

**Reply by Dreamsorcerer** [reply]
These tools don't look useful for large projects. They appear to just scrape the web pages, but large projects have 100,000s of dependents, and would therefore require 10,000s of requests. This really needs to be supported by Github.

**Comment by millievn** [comment]
so there is still no official support for this feature after three years?

**Comment by shapkarin** [comment]
Didn't know about much community support!
I'm sure that it's possible to implement even for a large amount of data with a good fullstack work.

**Comment by capricorn86** [comment]
I have created a simple tool that can sort dependents by stars:
https://github.com/capricorn86/happy-github-dependents
